03_letters/starfallz/main.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprint(out, strings.Join(sortLetters(letters("aba")), "\n"))
+}
+
+func letters(s string) map[rune]int {
+	letterCount := make(map[rune]int)
+
+	for _, currentLetter := range s {
+		letterCount[currentLetter]++
+	}
+
+	return letterCount
+}
+
+func sortLetters(m map[rune]int) []string {
+	allKeys := make([]rune, 0, len(m))
```

Comment:
I believe `len: 0` and `cap: len(m)` should be replaced with  `len: len(m)`. You don't plan to grow slice, you already know the size.

Reply:
Yeah you are right I am not sure why I did this. I will try it and see if that works or not. Thanks